pep-0689.rst

```diff
@@ -1,12 +1,13 @@
 PEP: 689
 Title: Unstable C API tier
 Author: Petr Viktorin <encukou@gmail.com>
+Discussions-To: https://mail.python.org/archives/list/python-dev@python.org/thread/PQXSP7E2B6KNXTJ2AERWMKKX42YP5D6O/
 Status: Draft
```

Comment:
This is not on the top of my TODO list. it'd be better to mark it deferred until I get to it again.
```suggestion
Status: Deferred
```
But I still hope to pick it up for `Python-Version: 3.12` :)

Reply:
Thanks! Applied.